projects/origin-stable/release.yaml

```diff
@@ -73,6 +73,10 @@ spec:
     from: {"kind":"DockerImage","name":"docker.io/openshift/origin-release:nodejs-8-browser-tests"}
     referencePolicy: {"type": "Local"}
     importPolicy: {"scheduled": true}
+  - name: tectonic-console-builder-v22
+    from: {"kind":"DockerImage","name":"quay.io/coreos/tectonic-console-builder:v22"}
+    referencePolicy: {"type": "Local"}
+    importPolicy: {"scheduled": true}
```

Comment:
I'd suggest we add this and the daemonset change in one PR, then actually update the jobs to use the new builder separately. That way we can use rehearse to test the jobs, and we don't risk breaking CI. (Rehearse will fail with this PR because the image stream tag isn't synced yet.)